service/pebble/plan/plan.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// This file's contents is directly copied from
+// github.com/canonical/pebble/internal/plan
+// Do not modify this file instead propose changes to Pebble and copy back.
```

Comment:
Grammar here:
```
Do not modify this file. Instead, propose changes to Pebble and copy the new version.
```

Also, how about listing the version / Git SHA that this was copied from (5842ea68c9c7)